templates_ark/example_cell_clustering.ipynb

```diff
@@ -673,6 +673,31 @@
     "    base_dir, cell_table_path, cell_data_name\n",
```

Comment:
<p>I think this path will need to be changed to the version pointing to Master. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>